server/neptune/workflows/internal/terraform/job/runner.go

```diff
@@ -22,7 +22,7 @@ const (
 // ExecutionContext wraps the workflow context with other info needed to execute a step
 type ExecutionContext struct {
 	Path      string
-	Envs      map[string]string
```

Comment:
I'm still wrapping my head around temporal versioning but is it safe to change this struct here since it gets passed into our legacy way of loading env variables? If a running workflow had to rerun a Plan operation, wouldn't this change cause non-determinism or do we only care about it when we orchestrate an activity?

Reply:
Yeah it's safe.  The only changes we need to be worried about are anything that gets serialized to the event log.  Usually those are activities or temporal constructs like `workflow.Sleep`, `workflow.Go` etc.